jenkins-pipelines/ics_space_amplification_test.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
```

Comment:
since this is not a longevity test, maybe would be useful to prefix this filename with features- or at least ICS (i.e with capital letters)

Reply:
done